src/models/LottoMachine.js

```diff
@@ -0,0 +1,17 @@
+import { Random } from '@woowacourse/mission-utils';
+
+import Lotto from './Lotto.js';
+import { LOTTO_PRICE_UNIT, REQUIRED_WINNING_NUMBERS_COUNT } from '../constants/lottoConstant.js';
+
+class LottoMachine {
+  generateLottos(purchasePrice) {
+    const lottoCount = purchasePrice / LOTTO_PRICE_UNIT;
+    const lottos = new Array(lottoCount)
```

Comment:
```suggestion
    const lottos = new Array.from({length: lottoCount}, () => new Lotto(...));
```
Array의 from을 사용하면 조금 더 간경하게 나타낼 수 있지 않을까 생각됩니다!

Reply:
Array.from() 메서드가 있었군요! 다음에는 이런 방식을 생각해보겠습니다.